gpflow/training/natgrad_optimizer.py

```diff
@@ -76,10 +78,11 @@ def minimize(self, model, var_list=None, session=None, feed_dict=None,
         opt = self.make_optimize_action(model, session=session, var_list=var_list, **kwargs)
         with session.as_default():
             for _i in range(maxiter):
-                opt()
+                result = opt()
```

Comment:
What does `opt()` return? Is that something useful and usable for callback?

Reply:
I just realised I didn't change the interface in the optimizer.py. That's where I should have documented the step_callback.

Basically we pass back whatever the optimiser returns. If anything, it will most likely be the values of trainable variables. So one can write a specialised task that having a specific optimiser in mind will do something with this returned value.